pkg/autoscaler/autoscaler.go

```diff
@@ -82,29 +87,32 @@ type totalAggregation struct {
 
 // Aggregates a given stat to the correct pod-aggregation
 func (agg *totalAggregation) aggregate(stat Stat) {
+	if stat.LameDuck {
+		// Drop stats from lameducked pod
+		return
+	}
```

Comment:
Do we need this if, given we're dropping in the server already?

Reply:
Outdated.